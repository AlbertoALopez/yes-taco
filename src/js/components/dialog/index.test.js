import React from 'react';
import Dialog from './index.js';

test('Dialog renders correctly', () => {
  const wrapper = shallow(
    <Dialog
      dialogText="Testing123"
      transitionId="testing-id"
    />);

  expect(wrapper).toMatchSnapshot();
});
