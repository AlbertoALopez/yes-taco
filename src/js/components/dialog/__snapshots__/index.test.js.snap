// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog renders correctly 1`] = `
<Col
  lg={12}
  xs={12}
>
  <Row
    center="xs"
  >
    <div
      className="dialog-container"
      id="testing-id"
    >
      <p
        className="dialog-text"
      >
        Testing123
      </p>
    </div>
  </Row>
</Col>
`;
